From 739ba3863b0aa520422ae63fd4c22057d9798b1b Mon Sep 17 00:00:00 2001
From: Eugene Krasnikov <k.eugene.e@gmail.com>
Date: Tue, 9 Apr 2013 17:44:55 +0200
Subject: [PATCH] Check if KEY_FILE is defined

Signed-off-by: Eugene Krasnikov <k.eugene.e@gmail.com>
---
 net/mac80211/debugfs_key.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/net/mac80211/debugfs_key.c b/net/mac80211/debugfs_key.c
index 1521cab..dd2517b 100644
--- a/net/mac80211/debugfs_key.c
+++ b/net/mac80211/debugfs_key.c
@@ -34,9 +34,12 @@ static const struct file_operations key_ ##name## _ops = {		\
 	.llseek = generic_file_llseek,					\
 }
 
+#ifdef KEY_FILE
+#undef KEY_FILE
 #define KEY_FILE(name, format)						\
 		 KEY_READ_##format(name)				\
 		 KEY_OPS(name)
+#endif
 
 #define KEY_CONF_READ(name, format_string)				\
 	KEY_READ(conf_##name, conf.name, format_string)
-- 
1.7.10

