From da3a47fdbbd0ffc511bcb38651ca9f7d87c05062 Mon Sep 17 00:00:00 2001
From: Eugene Krasnikov <k.eugene.e@gmail.com>
Date: Tue, 9 Apr 2013 18:17:26 +0200
Subject: [PATCH] Add ETH_P_802_3_MIN define

Signed-off-by: Eugene Krasnikov <k.eugene.e@gmail.com>
---
 include/net/cfg80211.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/net/cfg80211.h b/include/net/cfg80211.h
index bdba9b6..bef4141 100644
--- a/include/net/cfg80211.h
+++ b/include/net/cfg80211.h
@@ -60,7 +60,7 @@
  */
 
 struct wiphy;
-
+#define ETH_P_802_3_MIN 0x600
 /*
  * wireless hardware capability structures
  */
-- 
1.7.10

