From 8c57cbdebf214cf4022958116b28ab830d13dbb7 Mon Sep 17 00:00:00 2001
From: Eugene Krasnikov <k.eugene.e@gmail.com>
Date: Tue, 9 Apr 2013 18:26:03 +0200
Subject: [PATCH] Add wcn36xx to compat-drivers

Signed-off-by: Eugene Krasnikov <k.eugene.e@gmail.com>
---
 config.mk             |    1 +
 scripts/driver-select |    7 ++++++-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/config.mk b/config.mk
index d002f87..6828f79 100644
--- a/config.mk
+++ b/config.mk
@@ -1,3 +1,4 @@
+export CONFIG_WCN36XX=m
 ifeq ($(wildcard $(KLIB_BUILD)/.config),)
 # These will be ignored by compat autoconf
  export CONFIG_PCI=y
diff --git a/scripts/driver-select b/scripts/driver-select
index 862f43b..73232b2 100755
--- a/scripts/driver-select
+++ b/scripts/driver-select
@@ -32,7 +32,7 @@ PURPLE="\033[35m"
 CYAN="\033[36m"
 UNDERLINE="\033[02m"
 
-SUPPORTED_80211_DRIVERS="ath5k ath9k ath9k_ap ath9k_htc carl9170 ath6kl ar5523 wil6210 b43 zd1211rw rt2x00 wl1251 wl12xx brcmsmac brcmfmac mac80211_hwsim"
+SUPPORTED_80211_DRIVERS="ath5k ath9k ath9k_ap ath9k_htc carl9170 ath6kl ar5523 wil6210 b43 zd1211rw rt2x00 wl1251 wl12xx brcmsmac brcmfmac wcn36xx mac80211_hwsim"
 
 # b43 needs some more work for driver-select, the SSB stuff, plus
 # what if you update b44 but not b43? It will bust.
@@ -508,6 +508,11 @@ case $1 in
 		disable_staging
 		disable_var_01
 		;;
+	wcn36xx)
+		select_drivers	CONFIG_WCN36XX
+		disable_staging
+		disable_var_01
+		;;
 	mac80211_hwsim)
 		select_drivers		CONFIG_COMPAT_MAC80211_HWSIM
 		disable_staging
-- 
1.7.10

